core/src/main/scala/besom/internal/ResourceDecoder.scala

```diff
@@ -39,12 +41,14 @@ object ResourceDecoder:
               .decode(value, propertyLabel)
               .map(_.withDependency(resource))
               .tapBoth(
-                err => log.debug(s"failed to extract custom property '$propertyName' from '$value': '$err'"),
-                value => log.debug(s"extracted custom property '$propertyName' from '$value'")
+                err => log.debug(s"failed to extract custom property '$propertyName' from '${printer.render(value)}': '$err'"),
+                _ => log.debug(s"extracted custom property '$propertyName' from '${printer.render(value)}'")
```

Comment:
Wait, this does something different than before, now it prints the value as it was before decoding. Before it printed the result of decoding, didn't it?

Reply:
This error message is not correct, I have a correction in another branch.

The `value` in the `tapBoth` was shadowing the `value` in `map` above.
The decoded value will be full of outputs (the ones you've complained about).

Reply:
I've reverted the debug message to display the resulting output data, but changed the variable name to get rid of shadowing.

I've changed the printer to turn `besom.internal.Output@3b751a0c` into `Output(?)` in logs.